### ISSUE TITLE
Generate targets from OpenAPI/Swagger document

### ISSUE THREAD

**Issue by omussell** [nginxinc/rtapi#5]
It would be cool if rtapi could use an OpenAPI format document as input for a list of targets. Or alternatively have a flag on rtapi that parses an OpenAPI document and writes out rtapi compatible targets to an endpoints.json file.

**Comment by alessfg** [comment]
I agree, adding OpenAPI as an input type would be a nice new feature!
